core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -567,4 +567,14 @@ package object config {
       .intConf
       .checkValue(v => v > 0, "The value should be a positive integer.")
       .createWithDefault(2000)
+
+  private[spark] val BARRIER_SYNC_TIMEOUT =
+    ConfigBuilder("spark.barrier.sync.timeout")
+      .doc("The timeout in milliseconds for each barrier() call from a barrier task. If the " +
```

Comment:
will users set this config in milliseconds? I feel seconds should be more common.

Reply:
+1